Web/Edubase.Web.UI/Assets/Scripts/GiasVueComponents/DownloadCategorisedFields.vue

```diff
@@ -28,7 +31,20 @@ export default {
     hasError: Boolean,
   },
   components: {
-    checkbox
+    checkbox,
   },
-}
+  methods: {
+    updateSelectedFields(id, isChecked) {
+     if (isChecked) {
+       this.selectedFields.push(id);
+    } else {
+      const index = this.selectedFields.indexOf(id);
+      if (index > -1) {
+        this.selectedFields.splice(index, 1);
+	}
+      }
+	this.$emit('update:selectedFields', this.selectedFields);
+    },
+  },
+};
```

Comment:
I've not specifically tested/checked this.

A pointer to the pages this loads on would be appreciated :)

Reply:
I think this part was to do with the downloads - It had to be changed due to the way v-model works in Vue3 (so I switched from v-model to :checked and @update).
I have done a few tests in there and it appears to be working as expected.
<img width="848" alt="image" src="https://github.com/user-attachments/assets/c7250600-3e82-49dc-882b-aae1173f6caa">